src/components/Projects.tsx

```diff
@@ -1,7 +1,149 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/knife.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+import Background from "./assets/home-page/background.jpg";
+import {
+  color,
+  space,
+  typography,
+  TypographyProps,
+  ColorProps,
+  SpaceProps,
+} from "styled-system";
+
+type HeadingProps = SpaceProps & ColorProps & TypographyProps;
+
+const Header = styled.div<HeadingProps>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+`;
+
+const Title = styled.h1<HeadingProps>`
+  ${space}
+  ${color}
+  ${typography}
+  font-family: SangBleu OG Serif Light Regular;
+`;
+
+const IconContainer = styled.div`
+  width: 200px;
+  height: 120px;
+`;
+
+const Icon = styled.a`
+  a:hover {
+    position: relative;
+    width: 500px;
+    height: auto;
+  }
+`;
 
 const Projects = () => {
-  return <div>Projects</div>;
+  return (
+    <div
+      className="bg_image"
+      style={{
+        backgroundImage: "url(" + Background + ")",
+        backgroundSize: "cover",
+        height: "140vh",
+      }}
+    >
+      <Header>
+        <Link to="/" style={{ textDecoration: "none" }}>
+          <Title p={7} color="athensGray" fontSize={5}>
+            OXYMORE
+          </Title>
+        </Link>
+      </Header>
+      <IconContainer>
+        <Icon>
+          <img
+            src={stairs}
+            style={{
+              position: "relative",
+              top: 100,
+              left: 100,
+            }}
+          ></img>
+        </Icon>
+      </IconContainer>
+      <IconContainer>
+        <Icon>
+          <img
+            src={shell}
+            style={{ position: "relative", top: 240, left: 100 }}
```

Comment:
Can we use styled-components/styled-system for this?
```
const IconImg = styled.img`
  ${layout};
  ${position};
  ${space};
// not sure which of these you'll need!
`;

<IconImg 
  src="shell"
  top="240"
  left="100"
/>
```

Reply:
See above but yes 1000% =)